### DISCUSSION TITLE
Sending template and normal message - Error 131047

### DISCUSSION THREAD

**Discussion by filipemh** [netflie/whatsapp-cloud-api#105] [Q&A]
I'm using WABA API from the cloud and every time I run the stream process:

- Sending Template Message - OK (Sent and Received)
- Sending Normal Message - OK (Sent and Not Received - Error 131047)

In the normal message sending process, I get (webhook) error: 
131047 - The message was not sent because more than 24 hours have passed since the last response from the customer to this number

As per the documentation, I'm following right.

Is there anything I can do?
Or is it my production account?
Or does it really happen to everyone?

Has anyone been through this situation?

Filipe Holanda

**Comment by aalbarca** [comment]
Hi @filipemh 

We are not the developers of the WhatsApp API, we have only created a PHP wrapper for WhatsApp HTTP Cloud API. You must make your query to Facebook or check the Whatsapp Cloud API documentation.

Best regards,
Alex.

**Comment by avido** [comment]
https://www.facebook.com/business/help/2640149499569241
